homeassistant/components/system_bridge/sensor.py

```diff
@@ -51,88 +51,134 @@ class SystemBridgeSensorEntityDescription(SensorEntityDescription):
 
 def battery_time_remaining(data: SystemBridgeCoordinatorData) -> datetime | None:
     """Return the battery time remaining."""
-    if (value := getattr(data.battery, "sensors_secsleft", None)) is not None:
-        return utcnow() + timedelta(seconds=value)
+    if data.battery.time_remaining is not None:
+        return dt_util.utcnow() + timedelta(seconds=data.battery.time_remaining)
     return None
 
 
 def cpu_power_package(data: SystemBridgeCoordinatorData) -> float | None:
     """Return the CPU package power."""
-    if data.cpu.power_package is not None:
-        return data.cpu.power_package
+    if data.cpu.power is not None:
+        return data.cpu.power
     return None
```

Comment:
I mean, if `data.cpu.power` is `None`, what's the difference in just returning `data.cpu.power` instead of None. (as in, this function doesn't add much imo)

Reply:
Good point